Maya/cicd/batch/buildMaya2Babylon.bat

```diff
@@ -0,0 +1,60 @@
+@echo off
+rem
+rem  buildBabylon2Maya.bat  -- Build and install dot net plugin
+rem 
+rem  usage:  buildBabylon2Maya
+rem  where:  install dll and nll assemblies to 2021 plug-ins directory:
+rem          located at: C:\Program Files\Autodesk\Maya(Version)\bin\plug-ins
+rem
+
+if "%1" == "" goto nomayaver
+
+set mayaversion=%1
+set mayalocation=%ProgramFiles%\Autodesk\Maya%mayaversion%
+
```

Comment:
It is generally a good idea to put `setlocal` at the beginning of a batch script to prevent variable leakage.

Reply:
Same for the other batch script.